Array in Javascript/README.md

```diff
@@ -136,6 +219,37 @@ We would have to loop through the whole array (worst case) or until we find it:
 Full code can be found at [here](src/searching.js)
 ### Micro-Challenge
 Can you find more efficient solution ? Create a PR and add it here as Micro-Challenge solution
```

Comment:
Remove the following statement:
`Create a PR and add it here as Micro-Challenge solution`